packages/aragon-wrapper/src/utils/transactions.js

```diff
@@ -0,0 +1,110 @@
+import { toBN } from 'web3-utils'
+import { getAbi } from '../interfaces'
+
+const DEFAULT_GAS_FUZZ_FACTOR = 1.5
+const PREVIOUS_BLOCK_GAS_LIMIT_FACTOR = 0.95
```

Comment:
How did you arrive at this number? is it reduced as a safety margin?

Reply:
More or less just picked one. The actual possible decrease is only `previous limit / 1024`, which is quite small, but we generally want at least a small safety margin over the block gas limit anyway.